query-sql/src/lib.rs

```diff
@@ -172,6 +172,19 @@ impl QueryFragment for Op {
     }
 }
 
+macro_rules! interpose {
```

Comment:
No, I later use `interpose` wherever I need to produce a sequence of separated things, not just in `SourceAlias` — it's used four times by the end of this PR.